core/backend/src/ExportGraphics.ts

```diff
@@ -139,11 +139,19 @@ export interface ExportGraphicsOptions {
    * will not be supplied via onGraphics. See [IModelDb.exportPartGraphics]($core-backend)
    */
   partInstanceArray?: ExportPartInstanceInfo[];
-  /** Max distance from a face to the original geometry, see [StrokeOptions]($core-geometry) */
+  /** Max distance from a face to the original geometry, see [StrokeOptions]($core-geometry).
+   * If not supplied, defaults to zero and angleTol will control the quality of the resulting mesh.
+   */
   chordTol?: number;
-  /** Max angle difference in radians for approximated face, see [StrokeOptions]($core-geometry) */
+  /** Max angle difference in radians for approximated face, see [StrokeOptions]($core-geometry).
+   * If not supplied, defaults to PI/12 (15 degrees).
+   */
   angleTol?: number;
-  /** Max length of any edge in generated faces, see [StrokeOptions]($core-geometry) */
+  /** Max length of any edge in generated faces, see [StrokeOptions]($core-geometry).
+   * If not supplied, there is no maximum length of an edge. Supplying this value can greatly increase the
+   * size of the resulting geometry, and should only be done in cases where necessary (if you don't know
+   * that's it's necessary, it's almost certainly not!)
```

Comment:
`/s/that's/that/`

Reply:
Fixed, thanks.

Reply:
RE: consolidation... if I did it again, I'd make the tolerance options a separate child object. But I prefer the minor duplication here to making clients step into another interface when exploring the API. I find that pretty onerous for simple options in some of our other APIs.